.babel-preset.js

```diff
@@ -25,31 +24,31 @@ function preset(context, options = {}) {
   return {
```

Comment:
probably, originally this was this was that fake babelrc.js pattern, but we don't actually want it to be a rc file since other packages use it as a preset. I think i left the `.` simply to keep the files close together, we should probably move this it a real package tho

Reply:
Your call really. I get the point as far as keeping them together, I do tend to think dotfiles get confusing though.